common

```diff
@@ -23,17 +23,23 @@ function clip {
 
 # Route nano to 'no-trailing-newlines' mode
 function nano {
-    "$(which nano)" -L "${@/ /\ }"
+    "$(command -v nano)" -L "${@/ /\ }"
 }
 
 function setname {
     echo -ne "\033]0;${1}\007"
 }
 
 function get_latest_release() {
-  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-    grep '"tag_name":' |                                            # Get tag line
-    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub API
```

Comment:
Change `$1` to `${1:?}` to enforce its presence.

Reply:
Okay